### DISCUSSION TITLE
What day is it today?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#427] [Galaxy Trivia] [answered]
The day before two days after the day before tomorrow is Tuesday.

**Comment by peterbaccaro** [comment]
The answer is **Monday**.

1) **the day before tomorrow** is Today
2) **two days after** (**the day before tomorrow**) = Two days after (Today) = The day after tomorrow.
3) **The day before** (**two days after the day before tomorrow**) = The day before (The day after tomorrow)= Tomorrow
4) Tomorrow is Tuesday, what day is today? Answer is **Monday!**

**Comment by thuongtruong1009** [comment, marked as answer]
If the day before two days after the day before tomorrow is Tuesday, then today is Thursday.

Here's the reasoning:

"The day before tomorrow" is today + 1 day, which is Friday.
"The day before" Friday is Thursday.
"Two days after" Thursday is Saturday.
So, "the day before two days after the day before tomorrow" is Thursday, and we are told that this day is Tuesday.
Therefore, today must be two days after Tuesday, which is **Thursday**.
